App/src/pages/CourseBrowser.tsx

```diff
@@ -1,12 +1,14 @@
-import React from "react";
+import React, { useState } from "react";
 import Select from "react-select";
 import { Box, Button, Container, FormLabel, Heading, Stack } from "@chakra-ui/react";
-
+import { useNavigate } from "react-router-dom";
+import { Input } from "@chakra-ui/react";
 interface SubjectItem {
     value: string;
     label: string;
 }
 
+// when the api is implemented I will make the subjects available in the drop down
```

Comment:
What API endpoint are you waiting for? An endpoint to get all course subjects? This exists, it's `/Course/GetAllCourseSettings`, it's already being used by the add-course page